widget/widgetAssets/js/iocExtractionConfigService.service.js

```diff
@@ -62,10 +64,40 @@
             "key",
             "jSONValue"
           ]
+        },
+        globalVariablesToKeyStoreMapping: {
+          "sfsp-excludelist-ips": ["Excludelist_IPs", "IP Address", true],
+          "sfsp-excludelist-urls": ["Excludelist_URLs", "URL", true],
+          "sfsp-excludelist-domains": ["Excludelist_Domains", "Domain", true],
+          "sfsp-excludelist-ports": ["Excludelist_Ports", "Port", true],
+          "sfsp-excludelist-files": ["Excludelist_Files", "File", true],
+          "sfsp-excludelist-cidr-ranges": ["NA", "CIDR Range", true],
+          "sfsp-excludelist-file-hashes":["NA", "File Hash", true],
+          "sfsp-indicator-type-mapping": ["Indicator_Type_Map", "NA", true]
         }
       }
     }
 
+
+    function getIndicatorRegex() {
+      return executeAction('cyops_utilities', 'get_regx_of_indicators', null, []);
+    }
+
+
+    function executeAction(connector_name, connector_action, userLoginId, payload) {
+      return $resource(API.INTEGRATIONS + 'connectors/?name=' + connector_name)
+        .get()
+        .$promise
+        .then(function (connectorMetaDataForVersion) {
+          return connectorService.executeConnectorAction(connector_name, connectorMetaDataForVersion.data[0].version, connector_action, userLoginId, payload);
+        })
+        .catch(function (error) {
+          console.error('Error:', error);
```

Comment:
error should be meaningful like 'Error occurred while executing connector action [action name]. Error: ...'

Reply:
Working on this